docs_src/sql_databases/sql_app_py311/tests/test_sql_app.py

```diff
@@ -0,0 +1,46 @@
+from database import Base
+from fastapi.testclient import TestClient
+from main import app, get_db
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./test.db"
+
+engine = create_engine(
+    SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False}
+)
+TestingSessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+
+Base.metadata.create_all(bind=engine)
+
+
+def override_get_db():
+    try:
+        db = TestingSessionLocal()
+        yield db
+    finally:
+        db.close()
+
+
+app.dependency_overrides[get_db] = override_get_db
+
+client = TestClient(app)
+
+
+def test_create_user():
```

Comment:
can you please add unit tests for the other read user and item crud endpoints?

Reply:
I couldn't reliably reproduce those tests within the `docs_src/` dir, they exist within `tests/`.

Reply:
looking at all the other directories within `docs_src/`, they don't seem to belong there.

Reply:
I assumed they were probably meant/planned for another doc explaining how to create tests

Reply:
Found where the tests were being documented, `docs/advanced/testing-database.md` I'll have a look at it this week.